open_spiel/games/laser_tag.cc

```diff
@@ -481,8 +506,17 @@ void LaserTagState::ObservationTensor(int player,
   SPIEL_CHECK_GE(player, 0);
   SPIEL_CHECK_LT(player, num_players_);
 
+  if (fully_obs_) {
+    FullObservationTensor(values);
+  } else {
+    PartialObservationTensor(player, values);
+  }
+}
+
+void LaserTagState::FullObservationTensor(absl::Span<float> values) const{
```

Comment:
Add space between `const` and `{`. In general it's a good practice to run any PRs through cpplint (https://google.github.io/styleguide/cppguide.html#cpplint).

Reply:
Thanks for letting me known I'll make sure to do that for future PRs.